Assets/Scripts/CharacterControl/Character.cs

```diff
@@ -12,6 +12,9 @@ public class Character : MonoBehaviour
         private CharacterMode mode = CharacterMode.Human;
 
         private CharacterMovement movement;
+        public BoxCollider2D square;
+        public Sprite cat;
+        public Sprite human;
```

Comment:
Move these to the top (with the rest of the public fields), and maybe add the suffix `Sprite` so it's more clear.